.github/workflows/release.yml

```diff
@@ -0,0 +1,34 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - v*
+
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Ruby 2.6
+      uses: actions/setup-ruby@v1
+      with:
+        version: 2.6.x
+
+    - name: Build
+      run: |
+        printf -- "${GEM_PRIVATE_KEY}\n" > $HOME/certs/gem-private_key.pem
+        gem cert --add certs/gem-public_cert.pem
+        gem build kitchen-terraform.gemspec --silent
+      env:
+        GEM_PRIVATE_KEY: ${{ secrets.RUBYGEMS_PRIVATE_KEY }}
+
+    - name: Publish
+      run: |
+        gem push kitchen-terraform-*.gem
```

Comment:
Can we use the branch name rather than a wildcard character to protect against the upload of an accidentally committed gem file?

Reply:
I had copied the wildcard from the travis deployment but I think this is a good idea.

Reply:
`${GITHUB_REF#refs/tags/v}` will parse out the version from the tag